packages/babel-plugin/src/styled/__tests__/behaviour.test.tsx

```diff
@@ -72,7 +72,21 @@ describe('styled component behaviour', () => {
     expect(actual).toInclude('{font-weight:500}');
   });
 
-  it('should not pass down invalid html attributes to the node', () => {
+  it('should not destructure valid html attributes from props', () => {
+    const actual = transform(`
```

Comment:
Could you use the common `transform` from the `test-utils`?

Reply:
For the whole file or just this test? I'll give it a try.

Reply:
Tried using `import { transform } from '../../test-utils';` and everything died :(

Reply:
Right, I forgot it would affect non-snapshots too. Yeah, don't worry about it for now then.